doc/sphinx/yaml/general.rst

```diff
@@ -0,0 +1,92 @@
+.. highlight:: yaml
+
+*****************
+General Structure
+*****************
+
+Sections
+--------
+
+The top level of a Cantera YAML input file is a mapping which defines different
+input file sections. Each section consists of a list of mappings which define
+objects of the same type, e.g. reactions, species, phases, or elements. The
+``phases`` section of an input file contains all of the phase definitions.
+Multiple sections containing reaction, species, or element definitions can be
+used. The specific names ``reactions``, ``species``, and ``elements`` are used
+as defaults when looking for reactions, species, and elements to add to a phase.
+A simple input file has the following structure::
+
+    phases:
+    - name: gas
+      thermo: ideal-gas
+      additional: fields
+    - name: solid
+      thermo: model-name
+      additional: fields
+
+    species:
+    - name: A
+      additional: fields
+    - name: B
+      additional: fields
+    - name: C
+      additional: fields
+
+    reactions:
+    - equation: A + B <=> C + D
+      additional: fields
+    - equation: A + C <=> 2 D
+      additional: fields
+
+Units
+-----
+
+While Cantera generally works internally in SI units, input values can be
+provided using a number of different units.
+
+Compound units are written using the asterisk (``*``) to indicate
+multiplication, the forward slash (``/``) to indicate division, and the caret
+(``^``) to indicate exponentiation. Exponents can include negative and decimal
```

Comment:
Why use the caret rather than two asterisks?

Reply:
I was probably thinking in LaTeX. Caret is also used by Matlab and Excel. On the double asterisk side,  there's Python and Fortran. So... 🤷‍♂️. The single-character token is easier to parse, but that's not really a reason.

Reply:
Has this been changed/resolved?

Reply:
It has not been changed. There are plenty of languages which use each syntax, so I don't see a strong reason to prefer `**`.